Objects/exceptions.c

```diff
@@ -2862,6 +2862,51 @@ _PyErr_TrySetFromCause(const char *format, ...)
  * or minus, using the stream redirection syntax).
  */
 
+
+// Static helper for setting legacy print error message
+static int
+_set_legacy_print_statement_msg(PySyntaxErrorObject *self, Py_ssize_t start)
+{
+
+    PyObject *strip_sep_obj = PyUnicode_FromString("\r\n");
+    Py_UCS4 soft_space_check = PyUnicode_READ_CHAR(PyUnicode_FromString(","), 0);
```

Comment:
The result of PyUnicode_FromString(",") is not checked for NULL and leaked.

You don't need this. You can use just `soft_space_check = ','`. Actually I think the named variable is not needed.